yaml2sbml/yaml2sbml.py

```diff
@@ -105,19 +106,29 @@ def _create_compartment(model: sbml.Model):
 
 def _load_yaml_file(yaml_file: str) -> dict:
     """
-    Load YAML file to a dictionary.
+    Load YAML file from a dictionary.
 
     Arguments:
-        yaml_file: SBML model
+        yaml_file: directory YAML model
 
     Returns:
         yaml_dic: dictionary with parsed yaml file contents
+    Raises:
+        RuntimeError, if yaml can not be parsed, e.g. to incorrectly
+            formatted entries
     """
-    with open(yaml_file, 'r') as f_in:
-        yaml_contents = f_in.read()
-        yaml_dic = yaml.full_load(yaml_contents)
+    try:
+
+        with open(yaml_file, 'r') as f_in:
+            yaml_contents = f_in.read()
+            yaml_dict = yaml.full_load(yaml_contents)
 
-    return yaml_dic
+    except ScannerError:
+        raise RuntimeError('YAML file can not be parsed due to a Scanner '
```

Comment:
As also other causes besides "-" are possible, it would be good to just have the "minus" thing as an additional remark, but always return the ScannerError error message too.